features/support/before-hook.js

```diff
@@ -30,6 +31,18 @@ function promiseToWait(timeToWait) {
 }
 
 /**
+ * Generate a random string
+ * @method randomBytes
+ * @param {Number}      size    Number of bytes to generate
+ * @return {Promise}
+ */
+function randomBytes(size) {
```

Comment:
Would it be better to just give the new token a new name (`tiger2` or something like that) instead of calling an extra function/pulling in the `crypto` package?